content/en/containers/guide/autodiscovery-examples.md

```diff
@@ -0,0 +1,617 @@
+---
+title: "Autodiscovery: Scenarios & Examples"
+kind: guide
+further_reading:
+- link: "/agent/kubernetes/log/"
+  tag: "Documentation"
+  text: "Collect your application logs"
+- link: "/agent/kubernetes/apm/"
+  tag: "Documentation"
+  text: "Collect your application traces"
+- link: "/agent/kubernetes/prometheus/"
+  tag: "Documentation"
+  text: "Collect your Prometheus metrics"
+- link: "/agent/guide/autodiscovery-management/"
+  tag: "Documentation"
+  text: "Limit data collection to a subset of containers only"
+- link: "/agent/kubernetes/tag/"
+  tag: "Documentation"
+  text: "Assign tags to all data emitted by a container"
+---
+
+This page contains detailed example templates for configuring integrations in containerized environments in the following scenarios:
+
+- [Redis integration for all Redis containers](#redis-integration-for-all-redis-containers)
+- [Apache integration with HTTP Check for all Apache containers](#apache-integration-and-http-check-on-apache-containers)
+
+For more information about containers and integrations, see [Docker and Integrations][2] and [Kubernetes and Integrations][3].
+
+All examples make use of Datadog's Autodiscovery feature, which allows you to define configuration templates for Agent Checks on designated sets of containers. For more information about Autodiscovery, see [Getting Started with Containers: Autodiscovery][1].
+
+## Redis integration for all Redis containers
+
+In this example scenario, you have a containerized environment in which you want to set up and configure the [Datadog-Redis integration][5] for all containers that match the name `redis`. You may have one container named `redis` and another named `my-custom-redis`, and you want to configure the Redis integration for **both** containers.
+
+The Redis integration comes with [default auto-configuration][4], but you want to additionally specify a `password` parameter and configure log collection. 
+
+If you were to configure this integration **on a host**, you could reference [`redisdb.d/conf.yaml.example`][6] for parameters and create a `conf.yaml` file that contains the following:
```

Comment:
This is not a past tense, but a conditional tense. This page is about configuring on a container (not a host) BUT I want the reader to first imagine hypothetically what it would be like to configure it on a host and then transfer that over to the containerized use case. I'm going to keep the tenses and append the word "hypothetically" to the beginning of the sentence just to make my rhetorical technique more clear.